src/hooks/useDispatch.ts

```diff
@@ -1,9 +1,6 @@
 // eslint-disable-next-line no-restricted-imports
 import { useDispatch as _useDispatch } from "react-redux";
-import { ThunkDispatch } from "redux-thunk";
-import { RootActions } from "store/actions";
-import { RootState } from "index";
 
-export const useDispatch = () =>
-  _useDispatch<ThunkDispatch<RootState, void, RootActions>>();
-export type Dispatch = ReturnType<typeof useDispatch>;
+import { AppDispatch } from "store";
+
+export const useDispatch = () => _useDispatch<AppDispatch>();
```

Comment:
[note] Refactored to use properly exported types rather than needing to manually reconstruct them here ourselves.